packages/webpack/src/config/base.js

```diff
@@ -372,7 +372,18 @@ export default class WebpackBaseConfig {
     plugins.push(...(buildOptions.plugins || []))
 
     // Hide warnings about plugins without a default export (#1179)
-    plugins.push(new WarnFixPlugin())
+    plugins.push(new WarnFixPlugin(warn =>
+      !(warn.name === 'ModuleDependencyWarning' &&
+        warn.message.includes(`export 'default'`) &&
+        warn.message.includes('nuxt_plugin_'))
+    ))
+
+    if (_typescript.build && buildOptions.typescript && buildOptions.typescript.ignoreNotFoundWarnings) {
+      // NOTE: this needs to be added to the plugin stack before the `WebpackBar`
+      plugins.push(new WarnFixPlugin(warn =>
+        !(warn.name === 'ModuleDependencyWarning' && /export .* was not found in /.test(warn.message))
```

Comment:
You are pushing a 2nd plugin instance here. Wouldn't it be better to compose/alter the filter function instead of a second instance? 🤔 ☺️

Reply:
exactly. creating multiple instances is redundant...
I will change so that only one instance is enough 👍